test-utils/src/index.js

```diff
@@ -36,6 +37,7 @@ export function act(cb) {
 		rerender();
 	}
 
+	options.debounceRendering = previousDebounceRendering;
```

Comment:
Should `act` call `teardown` since it calls `setupRerender()`? `teardown` not only restores `debounceRendering` but also flushes any updates called so that the next test begins with an empty update queue.

Reply:
My thought is the same reading this. I think the setup and teardown processes should be symmetrical in `act`. Otherwise someone might add code to `setupRerender` in future and that wouldn't get undone at the end of `act`.